datadog/tests/data_source_datadog_rum_retention_filters_test.go

```diff
@@ -0,0 +1,37 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+const RumRetentionFiltersDataSourceTestAppId = "6d6b8210-efde-4a1d-bc4d-0f9773f21b0c"
+
+func TestAccRumRetentionFiltersDatasource(t *testing.T) {
+	t.Parallel()
+
+	_, _, accProviders := testAccFrameworkMuxProviders(context.Background(), t)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV5ProviderFactories: accProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: dataDatadogRumRetentionFilters(),
+				Check: resource.TestMatchResourceAttr("data.datadog_rum_retention_filters.testing_rum_retention_filters",
+					"retention_filters.#", regexp.MustCompile(`^(0|[1-9]\d*)$`)), // check the size of list is >= 0
```

Comment:
This doesn't really test much, but unfortunately we cannot guarantee what default retention filters will be there 😢